browser/src/slideshow/LayerRenderer.ts

```diff
@@ -47,10 +48,8 @@ class LayerRendererGl implements LayerRenderer {
 
 	constructor(offscreenCanvas: OffscreenCanvas) {
 		this.offscreenCanvas = offscreenCanvas;
-		this.gl = this.offscreenCanvas.getContext('webgl');
-		if (!this.gl) {
-			throw new Error('WebGL not supported');
```

Comment:
please leave this exception, I think we need that for detecting when to use 2d fallback

Reply:
Nice point

Reply:
@eszkadev I have tested, It works fine without it as we already throwing error from `RenderContextGl` constructor so we don't need to worry about it :)